lib/view/authentication/login_screen.dart

```diff
@@ -0,0 +1,253 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get/get.dart';
+import 'package:nscc_app/constants/buttons.dart';
+import 'package:nscc_app/constants/colors.dart';
+import 'package:nscc_app/constants/footer.dart';
+import 'package:nscc_app/constants/text_field.dart';
+import 'package:nscc_app/constants/text_styles.dart';
+import 'package:nscc_app/constants/validators.dart';
+import 'package:nscc_app/router/routes_names.dart';
+
+class LoginScreen extends StatelessWidget {
+  LoginScreen({super.key});
+  final TextEditingController _username = TextEditingController();
+  final TextEditingController _password = TextEditingController();
+  final GlobalKey<FormState> _formkey = GlobalKey<FormState>();
+  @override
+  Widget build(BuildContext context) {
+    final double width = MediaQuery.of(context).size.width;
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: GestureDetector(
+        onTap: () {
+          FocusManager.instance.primaryFocus!.unfocus();
+        },
+        child: Container(
+          alignment: Alignment.center,
+          padding: EdgeInsets.symmetric(
+            horizontal: 16,
+          ),
+          decoration: BoxDecoration(
+            gradient: AppColors.bgGradient,
+          ),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+              Container(),
+              Column(
+                children: [
+                  Container(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: 0,
+                      vertical: 35,
+                    ),
+                    decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(8),
+                      color: Color(0xffffffff).withOpacity(0.05),
```

Comment:
User AppColors class to user colors in ui

Reply:
Added colors from AppColors class